lib/rspec/retry.rb

```diff
@@ -54,7 +54,9 @@ def self.apply
             break if example.exception.nil?
 
             if exceptions_to_retry.any?
-              break unless exceptions_to_retry.include?(example.exception.class)
+              break unless exceptions_to_retry.any? do |exception_klass|
+                example.exception.class <= exception_klass
```

Comment:
would prefer `example.exception.is_a? exception_klass` but will get this merged either way